api-generation/src/main/scala/ai/h2o/sparkling/api/generation/python/ConfigurationTemplate.scala

```diff
@@ -80,8 +80,8 @@ object ConfigurationTemplate extends ((Array[Method], Array[Method], Class[_]) =
          |        return self
          |""".stripMargin
     } else {
-      s"""    def ${m.getName}(self, value):
-         |        self._jconf.${m.getName}(value)
+      s"""    def ${m.getName}(self, *args):
```

Comment:
Shouldn't the changes affect only the overloaded setters? Wouldn't variable arguments break auto-completion features in interactive python environments?

Reply:
Implementation was changed to use variable number of arguments just for overloaded setters. In other cases, parameter names of (original) Java methods were used for setter parameters. This is a slight difference to previous implementation where a generic parameter name `value` was utilized.